packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/TweenAction/TweenAction.tsx

```diff
@@ -34,10 +34,15 @@ const InterpolationMapOption: Record<string, string> = {
   [InterpolationType.EASEBOUNCE]: 'Ease in/out Bounce'
 }
 
+function parseDuration(value: string | number): string {
+  const duration = Number(value)
+  return duration > 0 && duration < 100 ? duration.toFixed(2) : duration.toString()
```

Comment:
not sure how long a tween could last, but validating the min length will work great